README.md

```diff
@@ -31,4 +31,4 @@ dotnet -p path/to/project test // not yet implemented
 
 ### More Information
 
-For more complete example usage, please see [Getting Started with xUnit.net and CoreCLR / ASP.NET 5](http://xunit.github.io/docs/getting-started-coreclr.html).
+For more complete example usage, please see [Getting Started with xUnit.net and DNX / ASP.NET 5](https://xunit.github.io/docs/getting-started-dnx.html).
```

Comment:
dnx is not the write link either. I believe we are missing an up to date documentation for xunit with dotnet CLI. It definitely isn't the dnx one.